src/api/goods.ts

```diff
@@ -10,10 +10,11 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods.sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
you will have questions about variables `a` and `b`. So, better change the names of them

Reply:
I'd say it's okay to call vars this simple in this case